src/modules/orderHelper.js

```diff
@@ -1,7 +1,8 @@
 import axios from 'axios'
 
+const authHeaders = JSON.parse(localStorage.getItem('userData'))
```

Comment:
why did you change it to const?

Reply:
Refactoring, this line was repeating 4 times in every function, so we made a separate const for it.

Reply:
Thanks, good job!